docs/developer-guide/ipxe-installer.md

```diff
@@ -30,29 +30,103 @@ image for the ISO installer, but cannot fit into an iPXE ramdisk root.
 ## Configuring an iPXE Server for a Triton Installation
 
 An iPXE installation needs to pull necessary files via HTTP/HTTPS for
-installation.  We provide a [tar
-archive](https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/ipxe-latest.tgz)
-that can be installed on a web server that an iPXE client can reach.
+installation.  We provide a [tar archive][triton-ipxe] that can be installed on
+a web server that an iPXE client can reach.
 
-The `triton-installer.ipxe` file will need modifications in its `testdomain`
-and `base-url` variables to match your deployment.  If your iPXE webserver is
-ipxe.example.com, and its directory URL is https://ipxe.example.com/triton,
-you would have these lines in triton-installer.ipxe:
-
-```
-	set testdomain ipxe.example.com
-	set base-url https://ipxe.example.com/triton
-```
-
-**NOTE** - The `testdomain` must allow ICMP echo requests and respond to them
-(aka. pings) so the installer can confirm reachability.
-
-The tar archive also includes a full tar archive of the [ISO
-installer](./docs/developer-guide/iso-installer.md) contents because an iPXE
-ramdisk image is not large enough to hold all of the Triton images required
-for an installation, and the iPXE installer must download the images after it
-has booted.
+The tar archive also includes a full tar archive of the
+[ISO installer][triton-iso-doc] contents because an iPXE ramdisk image is not
+large enough to hold all of the Triton images required for an installation, and
+the iPXE installer must download the images after it has booted.
 
 Using the example server above, once it has been setup, an iPXE client should
 access `https://ipxe.example.com/triton/triton-installer.ipxe` to install the
 Triton head node.
+
+## Using Joyent's Netboot server
+
+Joyent also provides a netboot server that you can use with iPXE to chain load
+the Triton installer and is kept up to date with the latest release. To install
+Triton using the iPXE installer, chain load the installer URL.
+
+    chain https://netboot.smartos.org/triton-installer/triton-installer.ipxe
+
+After provisioning your headnode, connect to the serial console to configure
```

Comment:
Do we know which tty device to which this defaults?  ISTR it's `ttyb`.  Should  that be explicitly documented?

Reply:
All of our ipxe stuff right now is just hard coded for ttyb. We could add a menu option to support toggling that like we do for the smartos one. (See https://github.com/joyent/smartos-netboot-server/blob/master/tools/render_ipxe_file.sh). That would be a change we need to make in sdc-headnode.

Reply:
Yeah, I'm putting that in.